src/fsharp/FSharp.Compiler.Service/FSharp.Compiler.Service.netcore.fsproj

```diff
@@ -0,0 +1,615 @@
+<Project Sdk="FSharp.NET.Sdk;
+Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <VersionPrefix>11.0.8</VersionPrefix>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <DefineConstants>
+      $(DefineConstants);
+      BUILDING_WITH_LKG;
+      COMPILED_AS_LANGUAGE_SERVICE_DLL;
+      COMPILER;
+      COMPILER_SERVICE;
+      COMPILER_SERVICE_ASSUMES_FSHARP_CORE_4_4_0_0;
+      EXTENSIBLE_DUMPER;
+      FSHARP_CORE_4_5;
+      FX_PORTABLE_OR_NETSTANDARD;
+      FX_ATLEAST_PORTABLE;
+      NETSTANDARD1_6;
+      PREFERRED_UI_LANG;
+      FX_EVENTWAITHANDLE_NO_IDISPOSABLE;
+      FX_NO_APP_DOMAINS;
+      FX_NO_ARRAY_LONG_LENGTH;
+      FX_NO_BEGINEND_READWRITE;
+      FX_NO_BINARY_SERIALIZATION;
+      FX_NO_CONVERTER;
+      FX_NO_CORHOST_SIGNER;
+      FX_NO_CUSTOMATTRIBUTEDATA;
+      FX_NO_DEFAULT_DEPENDENCY_TYPE;
+      FX_NO_EXIT_CONTEXT_FLAGS;
+      FX_NO_HEAPTERMINATION;
+      FX_NO_LINKEDRESOURCES;
+      FX_NO_PARAMETERIZED_THREAD_START;
+      FX_NO_PDB_READER;
+      FX_NO_PDB_WRITER;
+      FX_NO_REFLECTION_MODULE_HANDLES;
+      FX_NO_RUNTIMEENVIRONMENT;
+      FX_NO_SECURITY_PERMISSIONS;
+      FX_NO_SERVERCODEPAGES;
+      FX_NO_SYMBOLSTORE;
+      FX_NO_SYSTEM_CONFIGURATION;
+      FX_NO_THREAD;
+      FX_NO_THREADABORT;
+      FX_NO_WAITONE_MILLISECONDS;
+      FX_NO_WEB_CLIENT;
+      FX_NO_WINFORMS;
+      FX_NO_WIN_REGISTRY;
+      FX_REDUCED_CONSOLE;
+      FX_REDUCED_EXCEPTIONS;
+      FX_RESHAPED_CONSOLE;
+      FX_RESHAPED_GLOBALIZATION;
+      FX_RESHAPED_REFEMIT;
+      FX_RESHAPED_REFLECTION;
+      FX_RESHAPED_REFLECTION_CORECLR;
+      FSI_TODO_NETCORE;
+      INCLUDE_METADATA_READER;
+      INCLUDE_METADATA_WRITER;
+      NO_DEBUG_LOG;
+      NO_INLINE_IL_PARSER;
+      NO_LOADER_OPTIMIZATION;
+      NO_LOGGING_GUI;
+      RESHAPED_MSBUILD;
+      SIGNED;
+      STRONG_NAME_AND_DELAY_SIGN_FSHARP_COMPILER_WITH_MSFT_KEY;
+      TODO_REWORK_ASSEMBLY_LOAD;
+      TODO_REWORK_SERVER
+    </DefineConstants>
+    <NoWarn>
+      $(NoWarn);
+      44;
+      69;
+      65;
+      54;
+      61;
+      75;
+      62;
+      9;
+      2003;
+    </NoWarn>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <DelaySign>true</DelaySign>
+    <DebugType>portable</DebugType>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <!-- we set EnableDefaultEmbeddedResourceItems because I like the explicitness -->
+    <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
```

Comment:
here is missing `<AssemblyName>`, like that is adding `.netcore` suffix (because is by convention the file name)